Directory.Packages.props

```diff
@@ -1,5 +1,6 @@
 <Project>
   <ItemGroup>
+    <PackageVersion Include="Google.Apis.Storage.v1" Version="1.68.0.3604" />
```

Comment:
Please revert this change, edit apis.json to change the dependency version (in the block for Google.Cloud.Storage.V1) and run generateprojects.sh to regenerate the project file.

Reply:
Ok. I will revert this change and will edit apis.json with the same dependency version for (Google.Cloud.Storage.V1).

Reply:
@jskeet  I have reverted my changes from Directory.Packages.props and updated Google.Apis.Storage.v1 version (1.68.0.3604) in apis.json file and after running generateprojects.sh , Google.Apis.Storage.v1 project file got automatically updated.